src/raylib.h

```diff
@@ -1581,6 +1581,15 @@ RLAPI void DetachAudioStreamProcessor(AudioStream stream, AudioCallback processo
 RLAPI void AttachAudioMixedProcessor(AudioCallback processor); // Attach audio stream processor to the entire audio pipeline
 RLAPI void DetachAudioMixedProcessor(AudioCallback processor); // Detach audio stream processor from the entire audio pipeline
 
+#if defined(SUPPORT_EVENTS_AUTOMATION)
+//------------------------------------------------------------------------------------
+// Event Recording & Replaying
+//------------------------------------------------------------------------------------
+RLAPI void BeginEventRecording(bool displayText);
+RLAPI void StopEventRecording(const char *fileName);
```

Comment:
Following raylib API conventions, `BeginEventRecording()` should align with `EndEventRecording()`. But not sure if that's the best name for those functions...
Also, file IO is usually used on `Load*()` and `Export*()` functions. I always try to separate IO file loading from data processing logic. Actually, the API already has:
```c
#if defined(SUPPORT_EVENTS_AUTOMATION)
static void LoadAutomationEvents(const char *fileName);     // Load automation events from file
static void ExportAutomationEvents(const char *fileName);   // Export recorded automation events into a file
static void RecordAutomationEvent(unsigned int frame);      // Record frame events (to internal events array)
static void PlayAutomationEvent(unsigned int frame);        // Play frame events (from internal events array)
#endif
```
I need to think better about this API...